pkg/feature/api/feature.go

```diff
@@ -1162,9 +1165,39 @@ func (s *FeatureService) ArchiveFeature(
 		)
 		return nil, err
 	}
-	if err := validateArchiveFeatureRequest(req, features, localizer); err != nil {
-		return nil, err
+	var tgtF *domain.Feature
+	for _, f := range features {
+		if f.Id == req.Id {
+			tgtF = &domain.Feature{Feature: f}
+			break
+		}
```

Comment:
If the request ID is not in the index zero, it will return an internal.

Reply:
Thanks! fixed [4a00e23](https://github.com/bucketeer-io/bucketeer/pull/1076/commits/4a00e2383fe2ffba0e4e1033c3d485ed6439873d)